File: Sever/models/Supplier.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema

const SupplierSchema = new Schema({
    name: {
        type: String,
        required: true,
    },
    title: {
        type: String,
        required: true,
    },
    address: {
        type: String,
        required: true
    },
    contact: {
        type: String,
        required: true
    },
    bannerUrl: {
        type: String,
        required: true,
    },
    iconUrl: {
        type: String,
        required: true,
    }
})

module.exports = mongoose.model('suppliers', SupplierSchema)

File: OrdingFood/OrdingFood/Models/Supplier.swift
//
//  Supplier.swift
//  OrdingFood
//
//  Created by Vu Dat on 28/12/2021.
//

import Foundation
struct Supplier: Codable {
    var _id: String?
    var name: String?
    var title: String?
    var address: String?
    var contact: String?
    var bannerUrl: String?
    var iconUrl: String?
    
    enum Codingkeys: String, CodingKey {
        case _id,name,title,address,contact,bannerUrl,iconUrl
    }
}
